apollo-runtime-kotlin/src/commonMain/kotlin/com/apollographql/apollo/network/websocket/ApolloWebSocketNetworkTransport.kt

```diff
@@ -0,0 +1,18 @@
+package com.apollographql.apollo.network.websocket
+
+import com.apollographql.apollo.api.ApolloExperimental
+import com.apollographql.apollo.api.Response
+import com.apollographql.apollo.interceptor.ApolloInterceptorChain
+import com.apollographql.apollo.interceptor.ApolloRequest
+import com.apollographql.apollo.interceptor.ApolloRequestInterceptor
+import kotlinx.coroutines.flow.Flow
+
+@ApolloExperimental
+class ApolloWebSocketNetworkTransport(
+    private val apolloWebSocketFactory: ApolloWebSocketFactory
+) : ApolloRequestInterceptor {
```

Comment:
Feels weird to name it `NetworkTransport` if it inherits from `Interceptor`

Reply:
Sorry it was a mistake, it supposed to inherit `NetworkTransport`